index.js

```diff
@@ -7,7 +7,8 @@
   var xmlParser = require('xml2json');
   var FundamentalAccountingConcepts = require('./FundamentalAccountingConcepts.js');
 
-  function parse(pathToXbrlDoc) {
+  function parse(pathToXbrlDoc, output) {
+    output = (output=== null)? false : output;
     var self = this;
```

Comment:
would be better to do:
 `output = output ? false : output;`

to account for undefined as well;